README.md

```diff
@@ -1,53 +1,121 @@
+## [Hedera Open Source](#hedera-open-source)
 
-## Open Source
+This section features open source projects in the [Hedera Hashgraph](https://www.hedera.com) ecosystem. 
 
-* [Composer For Hedera Hashgraph](https://chrome.google.com/webstore/detail/composer-for-hedera-hashg/hdjnnemgikeoehneddegfcmkljenlean) - a community maintained tool to interact with Hedera from the Chrome environment.
-* [Exo-Demo](https://github.com/craigdrabiktxmq/exo-demo) - a demonstration of a web application communicating with a Swirld. The application tracks lions, tigers, and bears in a zoo. Users can add animals to the zoo and see the updated state. The zoo state is tracked in a Swirld, and adding an animal executes a transaction on the Swirld.
+**Software development kits (SDKs)**
+
+* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera Java SDK](https://github.com/hashgraph/hedera-sdk-java) - the official Java SDK for interacting with Hedera Hashgraph.
+* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera JavaScript SDK](https://github.com/hashgraph/hedera-sdk-js) - the official JavaScript SDK for interacting with Hedera Hashgraph. 
 * [Hedera .NET SDK](https://github.com/bugbytesinc/Hashgraph) - the community-maintained .NET SDK for interacting with Hedera Hashgraph.
 * [Hedera Go SDK](https://github.com/launchbadge/hedera-sdk-go) - the community-maintained Go SDK for interacting with Hedera Hashgraph.
-* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera Java SDK](https://github.com/hashgraph/hedera-sdk-java) - the official Java SDK for interacting with Hedera Hashgraph. Built and maintained by Hedera LLC.
-* ![official](https://i.imgur.com/y8jwGLa.png) [Hedera JavaScript SDK](https://github.com/hashgraph/hedera-sdk-js) - the official JavaScript SDK for interacting with Hedera Hashgraph. Built and maintained by Hedera LLC.
-* [Hedera Java Keygen Tool](https://github.com/hashgraph/hedera-keygen-java) - a command line utility for managing ED25519 key pairs for use with Hedera Hashgraph.
 * [Hedera Rust SDK](https://github.com/launchbadge/hedera-sdk-rust) - the community-maintained Rust SDK for interacting with Hedera Hashgraph.
-* [Hyperledger Mercury](https://github.com/hashgraph/hyperledger-mercury) - a community driven open source project geared at building a Hyperledger stack on top of the Swirlds platform.
-* ![official](https://i.imgur.com/y8jwGLa.png) [MyHbarWallet](https://github.com/hashgraph/MyHbarWallet) - a free, client-side interface helping you interact with Hedera Hashgraph.
 
-## Communities
+**Open source Hedera dev tools**
+
+* [Hedera Java Keygen Tool](https://github.com/hashgraph/hedera-keygen-java) - a command line utility for managing ED25519 key pairs for use with Hedera Hashgraph (deprecated).
```

Comment:
I'm pretty sure we aren't promoting the keygen tool anymore. I was actually going to archive the repo.

Reply:
That makes sense. If we archive the repository we can remove from this list, but otherwise, I was proposing adding here in hopes of getting someone to bring it back to life 😄